File: python/Larger_number.py
num1 = int(input("Enter first number : "))
num2 = int(input("Enter second number : "))
num3 = int(input("Enter third number : "))

if num1>=num2 and num1 >= num3:
    print(num1, " IS GREATER")
elif num2>=num1 and num2 >= num3:
    print(num2, " IS GREATER")
else:
    print(num3, " is greater")

File: java/Larger_number.java
import java.util.*;


public class Larger_number {
    public static void main(String[] arg){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first, second and third number : ");
       
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();
        int num3 = sc.nextInt();
        if(num1 >= num2 && num1 >= num3){
            System.out.print(num1 +" is greater");

        }else if(num2 >= num1 && num2 >= num3){
            System.out.print(num2 +" is greater");
        }else{
            System.out.print(num3 +" is greater");
        }
        sc.close();

    }
    
}
